slices/ca-certificates.yaml

```diff
@@ -15,6 +16,17 @@ slices:
         content.read(certs_dir + path) for path in content.list(certs_dir)
       ]
       content.write("/etc/ssl/certs/ca-certificates.crt", "".join(certs))
+      names = [
+        "mozilla/{}".format(name) for name in content.list(certs_dir)
+      ]
+      content.write("/etc/ca-certificates.conf", "\n".join(names))
+
+  tools:
```

Comment:
If there is a liberty for the slice naming, I'd suggest something more correspondant to the contents, like `full` or `fulldata`.

Reply:
> Yeah, since we are having the bundle at `/etc/ssl/certs/ca-certificates.crt`, and the symlink `/usr/lib/ssl/certs.pem -> /etc/ssl/certs/ca-certificates.crt`, it is possible for SSL clients to use the bundle rather than these single certificates for authentication. Since we are not able to run `openssl rehash` or `c_hash` during slicing, symlinking `/etc/ssl/certs/* -> /usr/share/ca-certificates/mozilla/*` is generally useless, as SSL clients don't use the `*.pem` certificates in `capath = /etc/ssl/certs` without them being rehashed.

With this, I don't think it would be necessary to call it `full` or `fulldata`.